redbot/core/bank.py

```diff
@@ -242,12 +242,19 @@ def _invalid_amount(amount: int) -> bool:
     if not isinstance(amount, int):
         raise TypeError("Withdrawal amount must be of type int, not {}.".format(type(amount)))
     if _invalid_amount(amount):
-        raise ValueError("Invalid withdrawal amount {} < 0".format(humanize_number(amount)))
+        raise ValueError(
```

Comment:
Wasn't too sure if this gets translated to users locale down the pipe never used the bot with a non en locale.
If it doesn't it make sense to override these to en_US, but if its translated later then this should be removed

Reply:
That's all good, you made the right call :+1: Even if these messages are sent to the user, they're not translated, so using `humanize_number` with a custom locale wouldn't make sense.